projects/storefrontlib/src/cms-components/misc/qualtrics/qualtrics.component.ts

```diff
@@ -1,12 +1,25 @@
 import { Component } from '@angular/core';
+import { QualtricsConfig } from './config/qualtrics-config';
 import { QualtricsLoaderService } from './qualtrics-loader.service';
 
+/**
+ * Adds the Qualtrics deloyment script whenever the component is loaded. The
+ * deployment script is loaded from the global configuration (`qualtrics.scriptSource`).
+ *
+ * The script is added when the route `NavigationEnd` occurs, so that the script is (re)added
```

Comment:
I believe this is a leftover from when the routing service was in play?

Reply:
indeed!